src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+
+    <div class="conteiner">
+      <form 
+        class="search__form"
+        data-qa="big"
+        action="/"
+        methot="get">
+        <label class="search__box">
+          <img
+            class="img"
+            src="images/Search.svg"
+            alt="magnifying glass"
+          >
+          <input 
+            type="text"
+            name="input"
+            class="search__input"
+            placeholder="Try &quot;Los Angeles&quot;"
+            data-qa="keypress"
+            required
+          >
+        </label>
+      </form>
+      <form 
+        class="search__form search__form-small"
+        data-qa="small"
+        action="/"
+        methot="get">
+        <label class="search__box">
+          <img 
+            class="img img-sm"
+            src="images/Search.svg" 
+            alt="magnifying glass">
```

Comment:
according to previous mentors recomendations
```suggestion
          <img 
            class="img img-sm"
            src="images/Search.svg" 
            alt="magnifying glass"
          >
```